subprojects/base-services/src/main/java/org/gradle/internal/operations/BuildOperationCategory.java

```diff
@@ -77,4 +79,8 @@ public boolean isGrouped() {
     public boolean isTopLevelWorkItem() {
         return topLevelWorkItem;
     }
+
+    public boolean isShowHeader() {
+        return showHeader;
+    }
```

Comment:
Can we be more descriptive (in doc or in field name) here? What is a _header?_ If this is only about transforms, then let's call that out here, too.